cli/package.json

```diff
@@ -60,6 +60,7 @@
     "semver": "^7.3.7",
     "tar": "^6.1.11",
     "tslib": "^2.4.0",
+    "xml-js": "1.6.11",
```

Comment:
Isn't there a more up to date library that can accomplish the same?
We already have `xml2js` which has not been updated in 3 years, but `xml-js` has not been updated in 4 years, so having more unmaintained dependencies will probably bring us problems in the future.